docs/sources/datasources/google-cloud-monitoring/preconfig-cloud-monitoring-dashboards.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Preconfigred dashboards"
+description = "Guide for using Google Cloud Monitoring in Grafana"
+keywords = ["grafana", "stackdriver", "google", "guide", "cloud", "monitoring"]
+aliases = ["/docs/grafana/latest/features/datasources/stackdriver", "/docs/grafana/latest/features/datasources/cloudmonitoring/"]
+weight = 10
++++
+
+# Preconfigured dashboards
+
+Google Cloud Monitoring data source ships with [out-of-the-box](#out-of-the-box-dashboards) and [curated](#curated-dashboards) dashboards. See also [Using Google Cloud Monitoring in Grafana]({{< relref "./_index.md" >}}) for detailed instructions on how to add and configure the data source.
+
+## Out-of-the-box dashboards
+
+Grafana v7.3 and later versions supports five of the most popular out-of-the-box GCP services. They are:
+
+- BigQuery
```

Comment:
Sorry Jita, I didn't know we already have the ## Out-of-the-box dashboards section. Out-of-the-box dashboards are the same thing as curated dashboards, so we need to remove the whole ##Out-of-the-box section.

Reply:
@sunker, thank you for the review. The original topic was so long, it was easy to overlook existing content. Let me know if you have any other comments/suggestions.